components/card/card.css

```diff
@@ -0,0 +1,246 @@
+@import url(/components/button/button.css);
+@import url(/components/badge/badge.css);
+
+:root {
+  --card-description-color: #808080;
+}
+
+.component-container {
+  display: flex;
+  justify-content: space-evenly;
+  padding: 1rem 0;
+}
+
+/* ||------ Standard Card start */
+
+.card {
+  height: max-content;
+  max-width: 300px;
+  border-radius: var(--large-border-radius);
+  box-shadow: 0 2px 7px rgba(0, 0, 0, 0.2);
+  position: relative;
+}
+
+/* Card Header */
+.card .card-header {
+  display: none;
+}
+
+/* Card image  */
+.card .card-body .card-image img {
+  object-fit: cover;
+  width: 100%;
+  height: 300px;
+  border-top-left-radius: var(--large-border-radius);
+  border-top-right-radius: var(--large-border-radius);
+}
+
+.card.card-small .card-image img {
+  height: 200px;
+}
+
+/* dismiss icon */
+.card .card-body .dismiss-icon {
+  color: var(--card-description-color);
+  display: none;
+}
+/* Card Badge */
+.card .card-body .card-icons {
+  display: none;
+}
+
+.card-icons .card-like-icon .icon:hover {
+  color: red;
+}
+
+/* Card Text Content */
+.card .card-body .card-text-content {
+  margin: 1rem 1.5rem;
+}
+
+.card .card-body .card-text-content .card-description {
+  color: var(--card-description-color);
+  margin: 0.5rem 0;
+}
+/* Card footer */
+.card footer.card-actions {
+  margin: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  column-gap: 1rem;
+}
+
+.card footer.card-actions .menu-icon {
+  margin-left: auto;
+  color: var(--velocity-black);
+  cursor: pointer;
+}
+
+/*------- || Standard Card ends */
+
+/* || Social Media card starts */
+
+.card.card-social-media .card-header {
+  display: flex;
+  align-items: center;
+  padding: 0.5rem;
+}
+.card.card-social-media .card-header .card-badge {
+  margin-right: 1rem;
+  margin-left: auto;
+  cursor: pointer;
+}
+
+.card.card-social-media .card-image img {
+  border-radius: 0;
+}
+
+.card.card-social-media .card-icons {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin: 0 1rem;
+}
+.card.card-social-media .card-icons .card-left-badge {
+  display: none;
+}
+
+.card.card-social-media .card-icons .card-badge .icon {
+  margin: 0 0.5rem;
+  cursor: pointer;
+  font-size: 2.2rem;
+}
+.card.card-social-media .card-body .card-title {
+  display: none;
+}
+.card.card-social-media .card-body .card-description {
+  color: var(--velocity-black);
+}
+
+.card.card-social-media .card-actions {
+  display: none;
+}
+/* || Social Media card ends */
+
+/* || Card with badges start */
+
+.card.card-w-badge .card-body .card-icons {
+  display: block;
+}
+
+.card.card-w-badge .card-icons .card-comment-icon,
+.card.card-w-badge .card-icons .card-share-icon {
+  display: none;
+}
+
+.card .card-body .card-icons .card-left-badge {
+  position: absolute;
+  top: 2rem;
+  left: -0.2rem;
+}
+.card.card-w-badge .card-body .card-icons .card-like-icon {
+  position: absolute;
+  top: 1.2rem;
+  right: 0.5rem;
+  color: gray;
```

Comment:
I see you have used variables for colours except for some places like this one. You can try to make variables wherever possible. Just a suggestion. Great work on the cards though!

Reply:
Okay got it i'll make the changes